### PR TITLE
new protocol directory

### PR DESCRIPTION
**Pull request by cpengilly**
create new `rollups` folder, update links

### CODE REVIEW

**Comment by cpengilly** [comment]
@sbvegan i think we can still merge since the link issues are with outside links (not internal)

File: pages/stack/protocol/_meta.json [hunk 1]
```diff
@@ -1,10 +1,5 @@
 {
-    "overview": "Rollup Protocol Overview",
-    "design-principles": "Design Philosophy & Principles",
-    "smart-contracts": "Smart Contract Overview",
-    "deposit-flow": "Deposit Flow",
-    "transaction-flow": "Transaction Flow",
-    "withdrawal-flow": "Withdrawal Flow",
+    "rollups": "Rollups",
```
Comment (sbvegan) [inline pages/stack/protocol/_meta.json:2]: "rollup": "Rollup"

File: pages/stack/protocol/rollups/_meta.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+    "overview": "Rollups Overview",
```
[the remark this answers is not in the document]
Comment (cpengilly) [reply pages/stack/protocol/rollups/_meta.json]: fixed

File: pages/stack/protocol/_meta.json (hunk 1 above)
Comment (cpengilly) [reply pages/stack/protocol/_meta.json]: fixed

**Review by sbvegan — APPROVED** [review]

**Pull request merged**